spec/serializers/api/v3/finance/statement_serializer_spec.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+module Api
+  module V3
+    module Finance
+      RSpec.describe StatementSerializer do
+        describe "serialization" do
+          let(:cohort) { create(:cohort, :next) }
+          let(:statement) { create(:ecf_statement, name: "January 2022", cohort:) }
+          let(:subject) { described_class.new(statement).serializable_hash }
+
+          it "returns the month" do
+            expect(subject[:data][:attributes][:month]).to eq("January")
+          end
+
+          it "returns the year" do
+            expect(subject[:data][:attributes][:year]).to eq("2022")
+          end
+
+          it "returns the cohort start year" do
+            expect(subject[:data][:attributes][:cohort]).to eq("2022")
+          end
+
+          it "returns the cut off dates" do
+            expect(subject[:data][:attributes][:cut_off_date]).to eq(statement.deadline_date.rfc3339)
+          end
+
+          it "returns the payment dates" do
+            expect(subject[:data][:attributes][:payment_date]).to eq(statement.payment_date.rfc3339)
+          end
```

Comment:
Could we collapse those attributes in one single test? ie. `have_including` or `have_attributes`

Reply:
If there was no logic attached to those maybe we could have, but since the logic differs in calculating those values my preference is to keep them separate in their own expectation so can test them under different scenarios and see if they fail